src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        String title = "fruit,quantity\n";
+        StringBuilder stringBuilder = new StringBuilder(title);
+        for (Map.Entry<String, Integer> entry : FruitStorage.fruitStorage.entrySet()) {
+            stringBuilder.append(entry.getKey()).append(",").append(entry.getValue()).append("\n");
+        }
```

Comment:
Consider checking if `FruitStorage.fruitStorage` is empty before iterating. If it's empty, you might want to return a message indicating that there are no fruits to report, or handle it according to your application's requirements.